Manatee.Trello/Rest/WebApiRequestProvider.cs

```diff
@@ -24,7 +24,7 @@ public IRestRequest Create(string endpoint, IDictionary<string, object> paramete
 					if (parameter.Key == RestFile.ParameterKey)
 					{
 						var rf = (RestFile)parameter.Value;
-						request.AddFile(parameter.Key, rf.ContentBytes, rf.FileName);
+						request.AddFile(parameter.Key, rf.FilePath, rf.FileName);
```

Comment:
We need to check which attachment method we're using here.  I don't want to completely replace the existing bytes method.